tests/daemon/test-daemon.c

```diff
@@ -1484,14 +1484,7 @@ static void
 test_daemon_refresh_and_reload_machine_id (Fixture       *fixture,
                                            gconstpointer  unused)
 {
-  g_autoptr(GFileIOStream) stream = NULL;
   g_autoptr(GError) error = NULL;
-  g_autoptr(GFile) override_machine_id_file =
-    g_file_new_tmp ("machine-id-override-XXXXXX",
-                    &stream,
-                    &error);
-  g_autofree gchar *override_machine_id_file_path =
-    g_file_get_path (override_machine_id_file);
```

Comment:
Why remove these? Shouldn't we be reading back the file and asserting it is exactly n hex characters long, with no hyphens?

Reply:
The purpose of this test isn't to check the written out machine-id file. It is to check that a different machine ID came through in the metrics request to the original one. It'd be worth adding a new test to check the format of the machine-ID that got written though.